pkg/views/workspace/create/summary.go

```diff
@@ -157,8 +152,26 @@ func projectDetailOutput(projectDetailKey ProjectDetail, projectDetailValue stri
 	return fmt.Sprintf("\t%s%-*s%s", lipgloss.NewStyle().Foreground(views.Green).Render(string(projectDetailKey)), DEFAULT_PADDING-len(string(projectDetailKey)), EMPTY_STRING, projectDetailValue)
 }
 
+func calculateViewportSize(content string) (width, height int) {
+	lines := strings.Split(content, "\n")
+	maxWidth := 0
+
+	for _, line := range lines {
```

Comment:
You can use `slices.Max(...)` here (or `MaxFunc()`)

Reply:
done